src/commands/asset-mgmt/setup.js

```diff
@@ -0,0 +1,125 @@
+const Command = require('../../base')
+const { flags } = require('@oclif/command')
+const renderShortDesc = require('../../utils/renderShortDescription')
+const fs = require('fs')
+
+const NETLIFY_GIT_LFS_SERVER_HOST = 'netlify-git-lfs.netlify.com'
+const NETLIFY_GIT_LFS_SERVER_PATH = '/.netlify/functions/lfs'
```

Comment:
I think we might want to proxy these URLs if they ever change.

The current URLs are proxied through https://github.com/netlify/netlifyctl/blob/master/site/_redirects#L4 those will move over to this repo once docs figures out how docs will work here

Reply:
@keiko713 let me know if you want me to setup that proxied redirect for ya

Reply:
this could indeed change. I may need to check if redirects will work as expected first. we will not merge this PR anytime soon, but should we add it to netlifyctl repo, or maybe we can wait until we move over it to cli repo?